butter_cms/client.py

```diff
@@ -26,11 +26,18 @@ def api_get(self, slug='', params=None, path_override=None):
         else:
             path = self.path
 
-        response = requests.get(
-            url=self.url + path + str(slug),
-            params=payload,
-            headers=headers,
-        )
+        try:
+            response = requests.get(
+                url=self.url + path + str(slug),
+                params=payload,
+                headers=headers,
+                timeout=10,
+            )
+        except requests.exceptions.Timeout:
+            raise Timeout()
```

Comment:
What's the reason for re-raising that error?

Did you want to make it explicit that we _expect_ this exception?
**If yes**, then you can simplify that:
```suggestion
        except requests.exceptions.Timeout:
            raise
```
The additional advantage is that if the original exception contained any valuable message, it will be preserved (as opposed to the current solution).

Reply:
Nice, thanks.